packages/eslint-config-base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@labkey/eslint-config-base",
-  "version": "0.0.11",
+  "version": "0.0.11-fb-add-linting.1",
```

Comment:
I'm not sure why the `package-lock.json` files were not checked in for the linting libraries -- so I added them. @labkey-alan is there a good reason why they were not?